### PR TITLE
Add Selva file to class repo

### LINKED ISSUE #38: xiscapericas tracks progress on Github Flow
### 1. Understanding the GitHub Flow 

- [x] Create a branch (git)
- [x] Add some commits (git) 
- [x] Open a pull request (GitHub) -- start a conversation about it 
- [ ] Additional commits (git) 
- [ ] Merge (git)

### PR DESCRIPTION
**Pull request by xiscapericas**
This PR adds Selva file to the class repo. This PR closes #38 

- @ruthtubau : how was the pizza? 
- Which flavour of ice cream do you prefer? 🍨

### CODE REVIEW

File: xiscapericas-selva.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Selva, Mallorca
```
Comment (asidi) [inline xiscapericas-selva.md]: @xiscapericas How do the locals look like? 🐊

Comment (ruthtubau) [inline xiscapericas-selva.md]: Betten than the sobrassada! 🍕

Comment (xiscapericas) [inline xiscapericas-selva.md]: @asidi Very very beauty, you already know that!!

**Comment by xiscapericas** [comment]
@asidi New info based on your comment on c7acf7e

**Comment by iservan** [comment]
👍 you can merge now 😛

**Pull request merged**